src/test/java/com/easypost/AddressTest.java

```diff
@@ -79,6 +79,10 @@ public void testCreateVerify() throws EasyPostException {
         assertInstanceOf(Address.class, address);
         assertTrue(address.getId().startsWith("adr_"));
         assertEquals("417 MONTGOMERY ST FL 5", address.getStreet1());
+
+        assertNotNull(address.getVerifications());
+        assertNotNull(address.getVerifications().getZip4().getErrors()); // Should have a error due to second line
```

Comment:
This test has empty errors, so technically we are not testing if the custom deserializer (AddressVerificationDeserializer) works or not. If it's hard to reproduce the error, we could define a JSON string and write a unit test in this file like what we have done in the past: https://github.com/EasyPost/easypost-java/blob/master/src/test/java/com/easypost/ErrorTest.java

Reply:
Sure, I originally had a test for a specific error code, but didn't want to lock it into API-specific expectations.

Reply:
Resolved